codex/conf.nim

```diff
@@ -293,6 +293,22 @@ type
         name: "validator-max-slots"
       .}: int
 
+      validatorPartitionSize* {.
```

Comment:
I was trying to find some good name (you can check the original issue, where there is some discussion about it, maybe you did not see it). I am also fine with `bucket` or `group` (`bucket` sounds very Amazonish though). So, `group` or `bucket`? We can also discuss shortly tomorrow.

> I also think "size" is a bit misleading here.

Yes, same here. Indeed, `size` parameter is a bit indirect. By saying how many partitions will be considered, it indirectly indicates the size of the partition. What would be your recommendation for the name of this param?

> the value should be a power of 2

You're right. On the other hand, by limiting it to powers of two, we limit the number of options available to node operator.

Reply:
> Something like --validator-groups for "size" and --validator-group-index for the assigned bucket.

Looks good to me.

Reply:
> You're right. On the other hand, by limiting it to powers of two, we limit the number of options available to node operator.

We can probably keep this value open without limiting it to a power of two. We can add a description in the CLI help. Most likely the only users to set this value will be us :) After that, users who use it would have a good understanding of what they're doing (hopefull). IOW, don't spend the time on this, as I don't think it's needed.